bin/check-ftp.rb

```diff
@@ -63,6 +63,10 @@ class CheckFTP < Sensu::Plugin::Check::CLI
          short: '-t SECS',
          proc: proc(&:to_i),
          default: 15
+  option :implicit,
```

Comment:
I think `implicit_ftps` would be a better name for this option. It's more clear.